compiler/modules/nback/module.bal

```diff
@@ -53,7 +53,13 @@ public function buildModule(bir:Module birMod, *Options options) returns [llvm:M
     foreach int i in 0 ..< functionDefns.length() {
         bir:FunctionDefn defn = functionDefns[i];
         bir:FunctionCode code = check birMod.generateFunctionCode(i);
-        check bir:verifyFunctionCode(birMod, defn, code);
+        bir:InvalidError|err:Semantic? verifyErr =  bir:verifyFunctionCode(birMod, defn, code);
+        if verifyErr is bir:InvalidError {
+            panic(verifyErr);
```

Comment:
This won't produce a panic with intelligible locations. We either need to pass it up or turn the locations into something more intelligible here.

Reply:
Does this mean that we should change the location entry in the InvalidError here before panicking here?

Reply:
Does this mean we should change the location entry in the InvalidError before panicking here?

Reply:
It's means only that the current code is not satisfactory.

Reply:
We can discuss in the meeting